draft-ietf-taps-impl.md

```diff
@@ -379,7 +379,13 @@ Each approach is appropriate in different use-cases and branch types. However, t
 
 The timing algorithms for racing should remain independent across branches of the tree. Any timers or racing logic is isolated to a given parent node, and is not ordered precisely with regards to other children of other nodes.
 
-### Delayed Racing
+
+### Immediate
+
+\[MICHAEL: This seems like an easy thing to do, and its description may be obvious, but I don't think we can skip it -- it just looks odd to first introduce the three variants and then only describe two of them.]
```

Comment:
The reason for leaving out immediate is that we recommend not using it, `implementations should not use immediate racing`.

Reply:
Understood. I removed it.